### PR TITLE
Update 03FocusOrder.md

### PR DESCRIPTION
**Pull request by kengdoj**
Issue #225 and Issue #210 and Issue #216

### CODE REVIEW

**Comment by AnnMarieDavis** [comment]
If you remove the reference to TAB and arrow once to revealed content, shouldn't the [SHIFT+TAB] and arrow key reference for navigating backwards also be removed?

File: docs/03FocusOrder.md [hunk 1]
```diff
@@ -33,7 +34,7 @@ Keyboard accessible interface components (links, form fields, drop down menus, s
 ### Test Instructions
 1.  Use the keyboard to navigate through interface components.
     1.  Use the keyboard to activate trigger controls that reveal hidden content (menus, dialogs, expandable tree list, etc.).
-        1.  Check that focus is moved to the revealed content. (It is acceptable to TAB once or use an arrow key to move the focus forward into the revealed content.)
+        1.  Check that the revealed focusable content is included in the focus order.
```
Comment (cliffbert) [inline docs/03FocusOrder.md:37]: "... in a sequence which is logical to the operation of the page" or similar?

Comment (AnnMarieDavis) [reply docs/03FocusOrder.md:37]: I like Cliff's suggestion.

Comment (kengdoj) [reply docs/03FocusOrder.md:37]: It's not shown in this diff view, but the next instruction covers that:

> 2. Check that the focus order preserves the meaning and usability of the page.

**Comment by kengdoj** [comment]
Working group reviewed during meeting.

**Pull request merged**